Lib/symtable.py

```diff
@@ -237,6 +237,12 @@ class Class(SymbolTable):
     def get_methods(self):
         """Return a tuple of methods declared in the class.
         """
+        import warnings
```

Comment:
Why not just put the import at the top of the file?

Reply:
To reduce import time.

Reply:
Right, but that import time is now spent on the method, and is no longer a one-time cost. I'll leave this for someone else to nitpick.